persistence/mediafile_repository_test.go

```diff
@@ -158,6 +158,31 @@ var _ = Describe("MediaRepository", func() {
 			Expect(mf.PlayCount).To(Equal(int64(1)))
 		})
 
+		It("preserves play date iff provided date is older", func() {
```

Comment:
typo: "...date _if_ provided..."

Reply:
Not a typo. "iff" means "if and only if", so the test checks both that it updates when the time is newer and that it doesn't when older.

Reply:
Sorry, but I rather have the test descriptions as plain English, so it is better to just write "if and only if".

Reply:
I think "iff" is a common abbreviation of "if and only if" (see e.g. [Wikipedia](https://en.wikipedia.org/wiki/If_and_only_if)) but if you insist then sure.

Reply:
Oh and it's actually an English word https://www.merriam-webster.com/dictionary/iff